### ISSUE TITLE
[OccHab] - Un module de saisie simple mais de synthèse complet, lié à un plugin QGIS ? 

### ISSUE THREAD

**Issue by Adrien-Pajot** [PnX-SI/GeoNature#1298]
Bonjour, 

Suite à une discussion dans le cadre de la mission COBIODIV SI avec Guillaume Coste d'ASTERS (@CGuillaume), quelques notions d'OccHab ont été abordées. 

D'une part, G. Coste est actuellement entrain de réaliser un plugin Python pour QGIS qui permettra de répondre au besoin des agents de terrain de la structure. Il réfléchit à pouvoir intégrer la table issue de ce plugin dans le module d'OccHab. 
En effet, OccHab peut permettre une saisie d'habitats simples mais ne possède pas toutes les fonctionnalités d'un logiciel SIG permettant de rentrer dans la complexité. 

De fait, il a plutôt été discuté des améliorations d'OccHab pour le côté synthèse. Par exemple, **gérer l'historisation des visites et des évolutions d'un habitat**. Il est possible d'imaginer que, sur un contour défini, un curseur puisse faire défiler la valeur de ces habitats dans le temps.
OccHab enrichissant son rôle de stockage et de centralisation des habitats, même complexes, pour un meilleur réemploi des données, des exports et des croisement avec d'autres modules. 

Le but de cette issue est de faire remonter les questions, les besoins et les propositions pour une meilleure gestion des données d'habitats dans GeoNature.

**Comment by DonovanMaillard** [comment]
Dans sa logique, occhab est destiné aux occurrences (relevés simples) d'habitats  comme occtax est destiné aux occurrences simples de taxons. 

Pourquoi ne pas plutôt enrichir le module de suivi pour qu'il puisse servir à la fois pour des suivis de taxons et des suivis d'habitats ? C'est même sans doute déjà faisable en s'appuyant sur les nomenclatures.

**Comment by Adrien-Pajot** [comment]
Pas tout à fait @DonovanMaillard, le module OccHab est à la fois module de saisie et de synthèse, contrairement à OccTax et Synthèse qui sont distincts pour les occurrences d’espèce. 

Cependant, l’idée de lier plusieurs types d’occurence est en effet discutée. Mais dans OccHab ? Dans un module dédié ? Dans la synthèse ?

**Comment by DonovanMaillard** [comment]
Oui, je parle bien de la partie "saisie avec des visites et un historique". 

Les données d'habitats, à termes, ne pourraient elles pas intégrer la synthèse tout comme les taxons? Au lieu d'avoir un cd_nom on a un cd_hab, au lieu d'avoir un nom latin on a un nom de milieu, mais pour le reste : localisation, observateur, date, déterminateur, certaines infos sont communes. Ou une "synthèse habitats" ? c'est sur qu'aujourd'hui GeoNature est essentiellement centré sur les taxons, mais peut être qu'il faut en effet chercher à ce qu'OccHab ne soit pas le seul outil pour les habitats.

**Comment by Adrien-Pajot** [comment]
Oui surtout que des modules comme le module Zones Humides, développé actuellement, est un module concernant des habitats qui pointerait vers OccHab

**Comment by camillemonchicourt** [comment]
Le module Occhab est un module de saisie d'occurrences d'habitats, mais aussi un module permettant de proposer une synthèse des occurrences d'habitat provenant de différentes sources d'habitats, de différents modules d'inventaire ou de suivi, ou d'imports de cartographies d'habitats faites en SIG ou autre.
